### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by xiaoxianBoy**
fix some typos

### CODE REVIEW

**Review by y0sh1ne — APPROVED** [review]

**Review by Rexogamer — APPROVED** [review]

**Review by offensive-vk — APPROVED** [review]
Nice catches @xiaoxianBoy !

**Review by lainiewoodyard22 — APPROVED** [review]
Check fixed

File: cli/src/async_pipe.rs [hunk 1]
```diff
@@ -227,7 +227,7 @@ impl hyper::server::accept::Accept for PollableAsyncListener {
 	}
 }
 
-/// Gets a random name for a pipe/socket on the paltform
+/// Gets a random name for a pipe/socket on the platform
```
Comment (yaokangyou) [inline cli/src/async_pipe.rs:230]: test

**Review by connor4312 — APPROVED** [review]

**Review by rzhao271 — APPROVED** [review]

**Review by Alimalik479 — COMMENTED** [review]
read

**Pull request merged**